app/helpers/contribution-status.js

```diff
@@ -0,0 +1,20 @@
+import Helper from '@ember/component/helper';
+import { inject as service } from '@ember/service';
+import { alias } from '@ember/object/computed';
+
+export default Helper.extend({
+
+  kredits: service(),
+  currentBlock: alias('kredits.currentBlock'),
+
+  compute([contribution]) {
```

Comment:
can this also be something on the model? (#rubythinking :)

Reply:
No, because you cannot inject services into a model.